docs/core/install/macos-notarization-issues.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Working with macOS Catalina Notarization
+description: How to handle notarization and certificate problems with macOS when you install the .NET Core runtime, SDK, and apps built with .NET Core.
+author: thraka
+ms.author: adegeo
+ms.date: 02/14/2020
+---
+
+# macOS Catalina Notarization and the impact on .NET Core downloads and projects
+
+Beginning with macOS Catalina (version 10.15), all software built after June 1, 2019, and distributed with Developer ID, must be notarized. This requirement applies to the .NET Core runtime, .NET Core SDK, and software created with .NET Core. This article describes the common scenarios you may face with .NET Core and macOS notarization.
+
+## Installing .NET Core
+
+The installers for .NET Core (both runtime and SDK) versions 3.1, 3.0, and 2.1, have been notarized since February 18, 2020. Prior released versions aren't notarized. You can manually install a non-notarized version of .NET Core by first downloading the installer, and then using the `sudo installer` command. For more information, see [Download and manually install for macOS](sdk.md?pivots=os-macos#download-and-manually-install).
+
+Beginning with the following versions, the .NET Core installers are notarized:
+
+- .NET Core Runtime
+  - 2.1.16
+  - 3.0.3
+  - 3.1.2
+
+- .NET Core SDK
+  - 2.1.512
+  - 3.0.103
+  - 3.1.102
+
+## appHost is disabled by default
+
+By default, non-notarized versions of the .NET Core SDK 3.0 and above produce a native Mach-O executable (known as the appHost) when your project compiles, published, and run. This executable is a convenient way to run your app. Otherwise, your app must be started by running `dotnet <filename.dll>`. When the appHost is enabled, the `dotnet run` command is invoked in the context of the appHost. For more information, see [Context of the appHost](#context-of-the-apphost).
```

Comment:
I'd recommend highlighting "appHost" here because you are defining it:

```suggestion
By default, non-notarized versions of the .NET Core SDK 3.0 and above produce a native Mach-O executable (known as the **appHost**) when your project compiles, published, and run. This executable is a convenient way to run your app. Otherwise, your app must be started by running `dotnet <filename.dll>`. When the appHost is enabled, the `dotnet run` command is invoked in the context of the appHost. For more information, see [Context of the appHost](#context-of-the-apphost).
```